app/src/androidTest/java/ch/epfl/sweng/SDP/DrawingActivityTest.java

```diff
@@ -24,4 +24,23 @@
     public void testCanvas() {
         onView(withId(R.id.paintView)).perform(click());
     }
+
+    @Test
+    public void tesDrawToggleIsClickable() {
+        onView(withId(R.id.fly_or_draw)).perform(click());
+        onView(withId(R.id.fly_or_draw)).check(matches(isClickable()));
```

Comment:
"onView(withId(R.id.fly_or_draw)).perform(click())" is useless, you can delete it